docs/DeveloperGuide.md

```diff
@@ -233,6 +233,12 @@ From the notes in the sequence diagram above, for every attribute in the instrum
 prompt to get user to provide information for that attribute. This is done through a series of methods in
 the `TextUi` class.
 
+### Edit instrument feature
+<img src="images/EditInstrumentSequenceDiagram.png" width="600"/>
+<img src="images/EditRefrence1.png" width="600"/>
```

Comment:
<img width="227" alt="ss6" src="https://user-images.githubusercontent.com/60163283/140518535-40f8b517-b34e-4c74-bae8-fe2f3fc9c759.PNG">
This shld be <<class>> as calling a static method of TextUi